bin/echo.c

```diff
@@ -226,10 +234,11 @@ int echo(struct s2n_connection *conn, int sockfd)
                             if (wait_for_event(sockfd, blocked) != S2N_SUCCESS) {
                                 S2N_ERROR_PRESERVE_ERRNO();
                             }
+                        } else {
+                            // Don't modify the counts if we were blocked from sending
```

Comment:
Since you put this comment on the positive side (we did send), I'd phrase it as "Modify the counts if we successfully sent". Otherwise without the context of this PR, the comment is pretty confusing .

Reply:
Oh yea hah, I will change that make sense.